.github/workflows/auto-build.yml

```diff
@@ -31,16 +31,24 @@ jobs:
           wget http://sealer.oss-cn-beijing.aliyuncs.com/auto-build/sealer.tar.gz
           sudo tar -xvf sealer.tar.gz -C /usr/bin
           sudo sealer login ${{secrets.PRIVATEWAREHOUSE}}
-          sudo sed -i "s/v1.20.14/$version/g" ./rootfs/etc/kubeadm.yml
-          sudo sed -i "s/v1.20.14/$version/g" ./rootfs/Metadata
+          sudo sed -i "s/v1.19.8/$version/g" ./rootfs/etc/kubeadm.yml
```

Comment:
Using `KUBE_VERSION` install v1.19.8

Reply:
Yes, kubernetes version is obtained in kubedm, kubedm.yaml is a fixed file uploaded to the OSS. Here, we only replace the contents of the file